custom_components/sat/coordinator.py

```
@@ -94,6 +100,29 @@ def device_state(self):
         """Return the current state of the device."""
         return self._device_state
 
+    @property
+    def device_status(self):
+        """Return the current status of the device."""
+        if self.boiler_temperature is None:
+            return DeviceStatus.INITIALIZING
+
+        if self.setpoint is None or self.setpoint <= MINIMUM_SETPOINT:
```

Ruff (0.8.2)

`MINIMUM_SETPOINT` may be undefined, or defined from star imports

(F405)